src/inputs/forms/json-schema-form/mock.ts

```diff
@@ -78,13 +78,25 @@ export const getDefaultSchema = () => ({
         },
         colors: {
             description: 'Favorite Colors',
-            title: 'Favorite Colors',
+            title: 'Favorite Colors (multi input)',
             type: 'array',
             prefixItems: [
                 { type: 'string' },
                 { enum: ['red', 'blue', 'gold', 'purple', 'gray'] },
             ],
-            default: ['gray', 'blue'],
+            default: ['gray'],
+            multiInputMode: true,
+        },
+        color: {
+            description: 'Favorite Color',
+            title: 'Favorite Color (single input)',
+            type: 'array',
+            prefixItems: [
+                { type: 'string' },
+                { enum: ['red', 'blue', 'gold', 'purple', 'gray'] },
+            ],
```

Comment:
suggestion: I strongly recommend to add description about `prefixItems` and `multiInputMode` keyword into storybook.

![image](https://user-images.githubusercontent.com/26986739/207240425-8d9bd223-f37d-48c4-a98c-49f2c06dbead.png)